File: PHP/Aksesoris.php
<?php
include 'Petshop.php';

class Aksesoris extends Petshop
{
    private $jenis;
    private $bahan;
    private $warna;

    public function __construct($id, $namaProduk, $stok_produk, $hargaProduk, $foto_produk, $jenis, $bahan, $warna)
    {
        parent::__construct($id, $namaProduk, $stok_produk, $hargaProduk, $foto_produk);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    public function setJenis($jenis)
    {
        $this->jenis = $jenis;
    }

    public function getJenis()
    {
        return $this->jenis;
    }

    public function setBahan($bahan)
    {
        $this->bahan = $bahan;
    }

    public function getBahan()
    {
        return $this->bahan;
    }

    public function setWarna($warna)
    {
        $this->warna = $warna;
    }

    public function getWarna()
    {
        return $this->warna;
    }

    public function __destruct()
    {
    }

}

File: Java/Aksesoris.java
class Aksesoris extends Petshop
{
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris()
    {
    }

    public Aksesoris(int id, String nama, int hargaProduk, int stokProduk, String jenis, String bahan, String warna)
    {
        super(id, nama, hargaProduk, stokProduk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public String getJenis()
    {
        return jenis;
    }

    public String getBahan()
    {
        return bahan;
    }

    public String getWarna()
    {
        return warna;
    }

    public void setJenis(String jenis)
    {
        this.jenis = jenis;
    }

    public void setBahan(String bahan)
    {
        this.bahan = bahan;
    }

    public void setWarna(String warna)
    {
        this.warna = warna;
    }
}